src/config/backpack/operations/list.php

```diff
@@ -64,4 +64,9 @@
     // Display the `Showing X of XX entries (filtered  from X entries)`?
     // Setting this to false will improve performance on big datasets.
     'showEntryCount' => true,
+
+    // In translatable cruds, when the user is viewing the list contents in a language that's not translated
+    // should Backpack show the original content in the default language or the empty value for the
+    // current untranslated language ? By default, we show the original content.
+    'useFallbackLocale' => true,
```

Comment:
Love that it's configurable. Though... I don't know if it's necessary at all, tbh. 

If EN is the fallback/default, I would NEVER expect the list/show operation to show "english-version" for Romanian/Portuguese. The way I see it, that is only something we did on the UpdateOperation to make it easier to translate. It would become a shit-show if we did that on List/Show... 

Why? Because afaik, when you use the model inside the app itself... it won't use the fallback locale. Right? 👀 If you do `$product->title` and that's not translated to PT... it will show... empty?! I'm asking!

Reply:
Oh damn! Excellent example! 👏👏👏 1000 words 

No, I do not expect to see empty stuff... but I don't quite expect to see them in the fallback language either... not without a notice or something that makes me understand "_RO version not available - here's the EN one_". But I don't like that option very much either, would further complicate how this works. Having a prefix like `RO missing. EN: _` would probably look like crap, right? 😔

Hmm... ok... I guess `useFallbackLocale()` is reasonable way to approach this then. Good find!

---

One more question so I understand this scenario completely... I (Cristian) have my admin panel in RO, I see the Articles translated in the EN (the fallback locale). I click Edit... what do I see? I expect Article with empty inputs right?

Mind shooting a quick video/loom/gif for me with the whole process, so I understand?